de/index.html

```diff
@@ -33,43 +33,42 @@ <h1>Frag nicht, ob du Fragen darfst, frag einfach</h1>
       <span class="name">Foobar123:</span>
       <p class="message">
         Gibt es hier irgendeinen Java-Experten, der mir bei meinem Problem, was
-        auch immer es ist, selbst wenn es nichts mit Java zu tun, helfen könnte -
-        oder könnte mir wenigstens irgendjemand antworten?
+        auch immer es ist, selbst wenn es nichts mit Java zu tun hat - und das
+        vielleicht von jemandem, der kein solcher Experte ist beantwortet werden
+        könnte - hilft?
       </p>
     </blockquote>
     <p>
-      Es gibt sehr viele Gründe weshalb Menschen, die wirklich das Wissen haben,
-      es nicht zugeben werden. Wenn du fragst, um etwas zu fragen, fragst du für
+      Es gibt sehr viele Gründe weshalb Menschen, die eigentlich das Wissen hätten,
+      es nicht zugeben werden. Wenn du fragst, um etwas zu fragen, fragst du nämlich nach
       mehr als du denkst.
     </p>
     <p>
       Du fragst Menschen, ob sie die Verantwortung für das Lösen deines Problems
-      übernehmen. Du forderst das Selbstvertrauen in die Fähigkeiten dieser Personen
-
-      heraus. Unnötigerweise schließt du andere Leute vom Beantworten deiner Fragen
-      aus. Relativ oft beantworte ich Fragen zu anderen Programmiersprachen oder
-      Libraries die ich noch nie benutzt habe, weil die Antworten (aus der Sicht
+      übernehmen. Du forderst das Vertrauen in die eigenen Fähigkeiten dieser Personen
+      heraus. Unnötigerweise schließt du andere Leute von der Beantwortung deiner Fragen
+      aus. Beispielsweise beantworte ich relativ oft Fragen zu anderen Programmiersprachen oder
+      Libraries die ich noch nie benutzt habe, weil die Antworten (zumindest aus der Sicht
       eines Programmierers) Allgemeinwissen sind.
     </p>
     <p>
-      Andererseits kann das auch so interpretiert werden...
+      Andererseits bedeutet eine solche Frage auch...
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        Ich habe eine Frage zu Java, aber zum Formulieren der Frage bin ich zu faul,
-        außer es gibt hier jemanden, der eventuell bereit wäre zu antworten.
-
+        Ich habe eine Frage zu Java, aber ich habe keine Lust, die Frage zu formulieren,
+        es sei denn, es gibt hier jemanden, der eventuell bereit wäre zu antworten.
       </p>
     </blockquote>
     <p>
-      ...was einfach nur faul aussieht. Wenn du nicht gewillt bist deinen Beitrag
+      ...was einfach nur nach Faulheit aussieht. Wenn du schon nicht gewillt bist deinen Beitrag
       zum Beantworten deiner Frage beizutragen, warum sollten wir das?
     </p>
     <p>
-      Die Lösung ist nicht zum Fragen zu fragen, sondern einfach zu fragen. Es ist
-      unwahrscheinlich, dass jemand der nur ab in den Kanal guckt deine "Frage zum
-      Fragen" für interessant hält und tatsächlich auf deine Fragen antworten wird.
+      Die Lösung ist nicht zu fragen um zu fragen, sondern einfach zu fragen. Es ist
```

Comment:
The title says “Frag nicht um zu Fragen”, but this states “zu fragen um zu fragen” — shouldn't the second fragen be capitalized?

Reply:
Actually I think none of them should be capitalised, it's not a noun in that construction. "Das Fragen" is capital but "um zu fragen" shouldn't be. I'll take another look tomorrow.